HACKING.md

```diff
@@ -280,6 +281,8 @@ codebase to make sure the compilation is not broken, all tests pass and it's com
 refactorings, architectural changes or behavior changes introduced in Kubernetes. Make sure:
 
 1. `make clean ; hack/build-go.sh` compiles without errors and the standalone server starts correctly.
+1. all of our generated code is up to date by running all `hack/update-*` scripts.
+1. `hack/verify-open-ports.sh` runs without errors.
```

Comment:
Probably need something about the upstream unit test script that adds the required files.